style.css

```diff
@@ -0,0 +1,35 @@
+
+*{
```

Comment:
Hi @tchasinga I suggest you to set a font-family 👪
in your css code [click here for font-family ](@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap');)

Reply:
- [ ] Good job for suggest change the by providing the reference for lookup, but you didn't  includes an explanation, a "why", for the requested change.  
- For Example "Kindly I recommend you to set to  a font-family 👪 in your CSS code just like this `@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500,700&display=swap');`  because it allows you to control the visual appearance of the text on your website or application"